pdf2slides.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
+from pgmagick import Image, ImageList
+from wand.image import Image
+from GoogleAPIs import GoogleDrive, GoogleSlides
+import sys, os
+
+# If modifying these scopes, delete the file token.json
+SCOPES = [
+    "https://www.googleapis.com/auth/presentations",
+    "https://www.googleapis.com/auth/drive",
+]
+
+if not os.path.isfile("token.json"):
+    drive = GoogleDrive("token.json", "credentials.json", SCOPES)
+    slides = GoogleSlides("token.json", "credentials.json", SCOPES)
+    sys.exit()
+
+if len(sys.argv) < 2:
+    print("Syntax: " + sys.argv[0] + " <pdf file>")
+    sys.exit()
+
+pages = Image(filename=sys.argv[1], resolution=300)
+print("Read PDF file")
+
+drive = GoogleDrive("token.json", "credentials.json", SCOPES)
+slides = GoogleSlides("token.json", "credentials.json", SCOPES)
+print("Initialized APIs")
+
+# Create the presentation
+presentation = slides.createPresentation(sys.argv[1])
+pid = presentation["presentationId"]
+print("Created presentation " + pid)
+
+# Remove the blank first slide
+slides.clearSlides(presentation)
+print("Removed blank slide")
+
+# Populate the slides
+i = 0
```

Comment:
Maybe should start from 1?

Reply:
It's only for output during build, so doesn't affect created Slides document, but changed anyway :-)